tests/server/api.py

```
@@ -0,0 +1,34 @@
+    "image_url": test_image_url,
+    "should_evaluate": True,
+    "export_folder": "/test_export",
+    "messages": []
+}
+
+# Send the request
+print("Sending request to API...")
+response = requests.post("http://127.0.0.1:8000/", json=payload, stream=True)
+
+# Process the streaming response
+print("Receiving streaming response:")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)